packages/lab/src/list-next/ListItemNext.tsx

```diff
@@ -0,0 +1,75 @@
+import { forwardRef, HTMLAttributes } from "react";
+import { clsx } from "clsx";
+import { makePrefixer } from "@salt-ds/core";
+
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import listItemNextCss from "./ListItemNext.css";
+
+const withBaseName = makePrefixer("saltListItemNext");
+
+export interface ListItemNextProps extends HTMLAttributes<HTMLLIElement> {
+  label?: string;
+  /**
+   * If true, the particular list item in list will be disabled.
+   */
+  disabled?: boolean;
+  focused?: boolean;
+  selected?: boolean;
+  id?: string;
+  value: string;
+  role?: string;
+}
+
+export const ListItemNext = forwardRef<HTMLLIElement, ListItemNextProps>(
+  function ListItemNext(
+    {
+      children,
+      className: classNameProp,
+      disabled,
+      focused,
+      label,
+      role = "option",
+      selected,
+      id,
+      value,
+      onClick,
+      ...props
+    },
+    ref
+  ) {
+    const targetWindow = useWindow();
+    useComponentCssInjection({
+      testId: "salt-list-item-next",
+      css: listItemNextCss,
+      window: targetWindow,
+    });
+
+    const className = clsx(
```

Comment:
We should move this to be inline so we're consistent across the codebase. It also means we don't need to rename className to classNameProp